habitat_baselines/rl/ddppo/multi_node_slurm.sh

```diff
@@ -18,6 +18,7 @@ MASTER_ADDR=$(srun --ntasks=1 hostname 2>&1 | tail -n1)
 export MASTER_ADDR
 
 set -x
-srun python -u -m habitat_baselines.run \
+srun --ntasks=1 python -u -m habitat_baselines.run \
```

Comment:
Is this line change intentional

Reply:
Yeah, thought I pushed the commit reverting these before opening the PR :)